Description: Removes install_requires in order to avoid automatic package
 installation with some Python package managers.
Author: Josue Ortega <josue@debian.org>
Last-Update: 2023-07-02
Forwarded: not-needed

--- a/setup.py
+++ b/setup.py
@@ -540,6 +540,5 @@
     ]
     + graph_objs_packages
     + validator_packages,
-    install_requires=["tenacity>=6.2.0", "packaging"],
     zip_safe=False,
 )
